DB/sqlGenerator.sql

```diff
@@ -0,0 +1,129 @@
+create table "Semester"
+(
+    s_id   serial
+        constraint semester_pk
+            primary key,
+    s_term varchar(50) not null,
+    s_year varchar(10)
+);
+
+create table "Department"
+(
+    dept_id   integer      not null
+        constraint department_pk
+            primary key,
+    dept_name varchar(100) not null
```

Comment:
I think this can have a unique constraint

Reply:
dept_name is now a unique field